common/djangoapps/pipeline_mako/templates/i18n_helpers.html

```diff
@@ -0,0 +1,31 @@
+<%page expression_filter="h"/>
```

Comment:
@andy-armstrong I actually wasn't sure where this should live, so I'd be happy to move it somewhere else. Do you have any suggestions that would be appropriate? Maybe the lang_pref app?